api/catalog/api/models/mixins.py

```diff
@@ -47,26 +51,47 @@ class MediaMixin(models.Model):
     the work and info about the artist. Do not use this as the sole base class.
 
     The mixin adds
+
     - title: CharField
     - foreign_landing_url: CharField
     - creator: CharField
     - creator_url: CharField
+    - thumbnail: URLField
+    - provider: CharField
     """
 
-    title = models.CharField(max_length=2000, blank=True, null=True)
+    title = models.CharField(
+        max_length=2000,
+        blank=True,
+        null=True,
+        help_text="The name of the media.",
+    )
     foreign_landing_url = models.CharField(
         max_length=1000,
         blank=True,
         null=True,
         help_text="The landing page of the work.",
     )
 
-    creator = models.CharField(max_length=2000, blank=True, null=True)
-    creator_url = models.URLField(max_length=2000, blank=True, null=True)
+    creator = models.CharField(
+        max_length=2000,
+        blank=True,
+        null=True,
+        help_text="The name of the media creator.",
+    )
+    creator_url = models.URLField(
+        max_length=2000,
+        blank=True,
+        null=True,
+        help_text="A direct link to the media creator.",
+    )
 
     # Because all forms of media have a thumbnail for visual representation
     thumbnail = models.URLField(
-        max_length=1000, blank=True, null=True, help_text="The thumbnail for the media."
+        max_length=1000,
```

Comment:
We could add a note about `thumbnail` for image being generated by the thumbnail service, and the audio thumbnail being provided by the sources or being null.

Reply:
Added this information in 3d68558.